provider/cluster_rosa_classic_resource.go

```diff
@@ -347,6 +347,11 @@ func (t *ClusterRosaClassicResourceType) GetSchema(ctx context.Context) (result
 				Optional:   true,
 				Validators: proxyValidators(),
 			},
+			"default_ingress": {
```

Comment:
Please generate this attribute to the resource docs, by using [tfplugindocs](https://github.com/hashicorp/terraform-plugin-docs)

Reply:
It seems generate docs creates many files and changes their names, not relevant to my changes
`deleted:    docs/resources/identity_provider.md`
`docs/resources/rhcs_identity_provider.md`

Reply:
So please just add the relevant lines for your change, it need to be done semi manual